pkg/logflags/logflags.go

```diff
@@ -139,16 +159,15 @@ func writeListeningMessage(server string, addr net.Addr) {
 	if tcpAddr == nil || tcpAddr.IP.IsLoopback() {
 		return
 	}
-	logger := RPCLogger()
-	logger.Logger.Level = logrus.WarnLevel
+	logger := RPCLoggerWithLevel(logrus.WarnLevel)
 	logger.Warnln("Listening for remote connections (connections are not authenticated nor encrypted)")
 }
 
 func WriteError(msg string) {
 	if logOut != nil {
-		fmt.Fprintln(logOut, msg)
+		_, _ = fmt.Fprintln(logOut, msg)
```

Comment:
We don't do this `_, _ = fmt.Fprintln` thing anywhere.

Reply:
Ok.